### PR TITLE
Remove material-ui remnants from Detail component.

### PR DESCRIPTION
**Pull request by thealmightyone**

### CODE REVIEW

File: src/components/Table/Table.jsx [hunk 1]
```diff
@@ -31,26 +21,26 @@ class TableComponent extends Component {
   render() {
     const {schema, singular} = this.props.schema;
     return (
-      <Paper style={detailStyle}>
+      <div className={'pt-card pt-elevation-3'}>
```
Comment (sealionkat) [inline src/components/Table/Table.jsx]: We can use just `className="pt-card pt-elevation-3"`

File: src/components/Table/Table.jsx [hunk 2]
```diff
@@ -31,26 +21,26 @@ class TableComponent extends Component {
...
         <Breadcrumb breadcrumbs={[singular]}/>
         <TableToolbarComponent handleOpenModal={this.props.openModal} singular={singular}
           options={schema.propertiesOrder} properties={schema.properties}
           filterData={this.props.filterData} filterProperties={this.props.visibleColumns}
         />
 
-        <Table fixedHeader={true}>
-          <TableHeader>
+        <table className={'pt-table pt-interactive'} style={{width: '100%'}}>
```
Comment (sealionkat) [inline src/components/Table/Table.jsx]: The same as above

File: src/components/Table/TableToolbarComponent.jsx [hunk 3]
```diff
@@ -53,40 +47,44 @@ class TableToolbarComponent extends Component {
...
-          <TextField floatingLabelText={'Search'} value={this.state.filterValue}
-            onChange={this.handleSearchChange}
-          />
-        </ToolbarGroup>
-      </Toolbar>
+      <div>
+        <div className="pt-navbar-group pt-align-left">
+          <button type="button" className="pt-button pt-intent-primary pt-icon-add"
+            onClick={this.props.handleOpenModal}>{'New ' + this.props.singular}
+          </button>
```
Comment (sealionkat) [inline src/components/Table/TableToolbarComponent.jsx]: Use `Button` component.

File: src/components/Table/Table.jsx [hunk 4]
```diff
@@ -1,20 +1,10 @@
 import React, {Component, PropTypes} from 'react';
```
Comment (stepek) [inline src/components/Table/Table.jsx]: Table is a Detail Component ?
Please check commit message.

Comment (stepek) [inline src/components/Table/Table.jsx]: https://github.com/cloudwan/gohan_webui/pull/188

**Pull request merged**